content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/expiration_dates.rst

```diff
@@ -1,65 +1,273 @@
+checked.
 
-Product configuration
----------------------
+Once it is, a new :guilabel:`Expiration Date` checkbox appears that must also be clicked. When both
+are enabled, a new :guilabel:`Dates` field will appear to the right.
 
-Now, you can define different dates in the *inventory tab* of the product form:
+.. note::
+   If a product has stock on-hand prior to activating tracking by lots or serial numbers, an
+   inventory adjustment might need to be performed in order to assign lot numbers to the existing
+   stock.
 
-- Product Use Time: the number of days before the goods start deteriorating, without being
-  dangerous. This is used to calculate the Best before date on each lot/serial number received.
-- Product Life Time: the number of days before the goods may become dangerous and must not be
-  consumed. This is used to calculate the Expiration date on each lot/serial number received.
-- Product Removal Time: shows the number of days before the goods should be removed from the stock.
-  This is used to calculate the Removal date on each lot/serial number received.
-- Product Alert Time: refers to the number of days before an alert should be raised on the
-  lot/serial number. This is used to calculate the Alert date on each lot/serial number received.
-  Once the Alert date is reached, an Activity is assigned on the relevant lot/serial number to the
-  Responsible user defined on the Product.
+.. tip::
+   For processing large quantities of products on receipts or deliveries, it is recommended to
+   track :guilabel:`By Lots`, so multiple products can be traced back to the same lot, if any
+   issues arise.
 
-.. image:: expiration_dates/expiration_dates_02.png
+.. image:: expiration_dates/expiration-dates-product-configuration.png
    :align: center
+   :alt: Expiration dates configuration on the product form.
 
-Expiration Date on Lots/Serial Numbers
-======================================
+Under the :guilabel:`Dates` field, there are four categories of expiration information to
+configure for the product:
+
+- :guilabel:`Expiration Time`: the number of days after receiving products (either from a vendor or
+  in stock after production) in which goods may become dangerous and should not be used or
+  consumed.
+- :guilabel:`Best Before Time`: the number of days before the expiration date in which the goods
+  start deteriorating, *without* necessarily being dangerous yet.
+- :guilabel:`Removal Time`: the number of days before the expiration date in which the goods should
+  be removed from stock.
+- :guilabel:`Alert Time`: the number of days before the expiration date in which an alert should be
+  raised on goods in a particular lot or containing a particular serial number.
+
+.. note::
+  The values entered into these fields automatically compute the expiration date for goods entered
+  into stock, whether purchased from a vendor or manufactured in-house.
```

Comment:
Text within an admonition block needs to align with the first character of the admonition title, so `n` in `note` - the text here is one space short of alignment